flexget/plugins/input/couchpotato.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+import requests
+
+from flexget import plugin
+from flexget.event import event
+from flexget.entry import Entry
+
+log = logging.getLogger('couchpotato')
+
+class CouchPotato(object):
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'base_url': {'tpye': 'string'},
+            'port': {'type': 'number'},
+            'api_key': {'type':'string'}
+        },
+        'required': ['api_key', 'base_url', 'port'],
+        'additionalProperties': False
+    }
+
+    def on_task_input(self, task, config):
+    """Creates an entry for each item in your couchpotato wanted list.
+
+    Syntax:
+
+    couchpotato:
+      base_url: <value>
+      port: <value>
+      api_key: <value>
+
+    Options base_url, port and api_key are required.
+    """
+        url = '%s:%s/api/%s/movie.list?status=active' % (config['base_url'], config['port'], config['api_key'])
+        json = task.requests.get(url).json()
+        entries = []
+        for movie in json['movies']:
+            if movie['status'] == 'active':
+              title = movie["title"]
+              entry = Entry(title = '%s' % (title),
```

Comment:
We aim to have all new code PEP8 compliant, which states that keyword arguments shouldn't have any spaces ie. it should be `title='%s'`. Is it even necessary to do it like that? Wouldn't it work with `title=title`?